databricks/koalas/frame.py

```diff
@@ -4250,7 +4261,7 @@ def to_spark(self, index_col: Optional[Union[str, List[str]]] = None):
 
     to_spark.__doc__ = SparkFrameMethods.__doc__
 
-    def to_pandas(self):
+    def to_pandas(self) -> "pd.DataFrame":
```

Comment:
I don't think we need to surround with double-quotes.

Reply:
`dev/reformat` seem to change `'pd.DataFrame'` to double quotes after each run.
Shall we disable it?

Reply:
ah, sorry, I mean `def to_pandas(self) -> pd.DataFrame:` should be fine?

Reply:
Sorry, I misunderstood. Updated. It works great!